### PR TITLE
added table with data and queries

### PR DESCRIPTION
**Pull request by AhmedNazirMusah**
On this milestone:
- I create a database named vet_clinic.
- Created a table named animals with the following columns:
id: integer
name: string
date_of_birth: date
escape_attempts: integer
neutered: boolean
weight_kg: decimal
- Copied the SQL used in creating the table into the schema.sql
- Added all the required data and added the SQL used to the data.sql 
- The following are the required queries and their respective screenshots.
1. Find all animals whose name ends in "mon".

select all names with mon

2. List the name of all animals born between 2016 and 2019.

name of animale born between 2016 and 2019

3. List the name of all animals that are neutered and have less than 3 escape attempts.
select name from animals where neutered is t and ea is less than 3

4. List the date of birth of all animals named either "Agumon" 
select DOB if the name is Agumon or pikachu
or "Pikachu".

5. List the name and escape attempts of animals that weigh more than 10.5kg
name and escape attempt where weight is less than 10 5

6. Find all animals that are neutered.
all animals where neutered is true

7. Find all animals not named Gabumon.
select all animal where name is no Gabumon

8. Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
last one

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi @AhmedNazirMusah 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has an outdated README file

- [ ]  Please check out the newly updated [Microverse README Template](https://github.com/microverseinc/readme-template) and use it to update your current README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
```
Comment (Munsa1) [inline schema.sql]: - [ ] ( optional ) You can make the `id` to be `not null`, also you could set is a primary key by adding `PRIMARY KEY (id)`

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,8 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon%';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '01/01/2016' AND '01/01/2019';
+SELECT * from animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
```
Comment (Munsa1) [inline queries.sql:9]: - [ ] Kindly add a query and screenshot that lists name and escape attempts of animals that weigh more than 10.5kg as this is a requirement.

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,8 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon%';
```
Comment (Munsa1) [inline queries.sql:3]: - [ ] Kindly add a query to find all animals with a weight between 10.4kg and 17.3kg as this is also part of the requirements

File: README.md [hunk 4]
```diff
@@ -1,48 +1,123 @@
...
+   This is the initial data structure for a vet clinic. Constitutes a table to store animals' information, insert some data into it, and query it.
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
```
Comment (Munsa1) [inline README.md]: - [ ] Kindly remove this detail as it is not relevant to this project.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @AhmedNazirMusah, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @AhmedNazirMusah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**